compiler/qsc_parse/src/lex/cooked.rs

```diff
@@ -150,7 +151,7 @@ pub(crate) enum TokenKind {
 impl Display for TokenKind {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
         match self {
-            TokenKind::Apos => f.write_str("`'`"),
+            TokenKind::AposIdent => f.write_str("`'`"),
```

Comment:
This is the token name used in error messages. I would be more descriptive, like:

```suggestion
            TokenKind::AposIdent => f.write_str("apostrophe identifier"),
```